### DISCUSSION TITLE
Turn on and off cameras with the API

### DISCUSSION THREAD

**Discussion by dimmanramone** [mrlt8/docker-wyze-bridge#738] [Q&A]
Hi,

Is it possible to turn on/turn off one camera via the API. I wanted to make an automation in Home Assistant that start/stops the cameras based on presence.

**Comment by mrlt8** [comment]
V2 of the bridge will have a way to enable/disable a stream via the API, but this will only apply to the stream from the bridge - the stream in the app will still be active as switching that on/off would require communicating with the wyze servers.

**Reply by dimmanramone** [reply]
Ok thanks for your answer, then I guess I will continue doing that with an IFTTT webhook :)
